api/governance/models.py

```diff
@@ -0,0 +1,121 @@
+import datetime
+from decimal import Decimal
+import django
+
+from django.db import models
+from django.db.models import Sum
+from django.shortcuts import reverse
+
```

Comment:
If you adopt the ArrayField implementation of category tags in the next comment.

```suggestion
from django.contrib.postgres.fields import ArrayField

```